src/openvic-simulation/pop/Pop.cpp

```diff
@@ -145,11 +353,15 @@ bool PopManager::load_pop_type_file(
 	Strata const* strata = nullptr;
 	PopType::sprite_t sprite = 0;
 	Good::good_map_t life_needs, everyday_needs, luxury_needs;
+	PopType::income_type_t life_needs_income_type = NONE, everyday_needs_income_type = NONE, luxury_needs_income_type = NONE;
+	fixed_point_t life_needs_income_weight = 0, everyday_needs_income_weight = 0, luxury_needs_income_weight = 0;
 	PopType::rebel_units_t rebel_units;
+	Pop::pop_size_t max_size = 0, merge_max_size = 0;
 	bool state_capital_only = false, demote_migrant = false, is_artisan = false, allowed_to_vote = true, is_slave = false,
 		can_be_recruited = false, can_reduce_consciousness = false, administrative_efficiency = false, can_build = false,
 		factory = false, can_work_factory = false, unemployment = false;
-	Pop::pop_size_t max_size = 0, merge_max_size = 0;
+	fixed_point_t research_points = 0, leadership_points = 0, research_leadership_optimum = 0, tax_efficiency = 0;
```

Comment:
I don't think tax_efficiency has any effect.
I'll check it later. If it is meaningless, we can log a warning (vanilla excluded).

Reply:
How would we detect if it was vanilla or not?

Reply:
In vanilla only bureaucrats have it with value 1.0

Reply:
tax efficiency determines how much the pop contributes to state admin efficiency (not national admin efficiency, the one that you can see in the province window that varies from state to state) which in turn determines crime fighting efficiency

Reply:
Thanks, I tested it and it works.
State efficiency from bureacrats = tax_eff * bureaucrat state population % / defines.country.MAX_BUREAUCRACY_PERCENTAGE